packages/@o3r/core/schematics/rule-factories/otter-environment/index.ts

```diff
@@ -116,21 +118,22 @@ export function updateOtterEnvironmentAdapter(
    * @param context
    */
   const generateEnvironmentFiles = (tree: Tree, context: SchematicContext) => {
+
     const workspaceProject = getProjectFromTree(tree, options.projectName, 'application');
     if (!workspaceProject) {
       context.logger.error('No application detected, the environment can not be generated');
       return tree;
     }
 
-    if (tree.exists('/src/environments/environment.ts')) {
+    if (tree.exists(nodePath.join(workspaceProject.root, '/src/environments/environment.ts'))) {
```

Comment:
why the initial `/` which is a bit misleading?

Reply:
It''s from copy paste. I'll remove it